config/environments/development.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
This line should not be uncommented. I know Devise told you to do it, but we're not using mailers.